Frontend/ui-customer/src/MerchantConfigs/DefaultConfig.purs

```diff
@@ -136,4 +136,5 @@ config =
       auto : true,
       cabs : false
     }
+  , enableContactSupport : true
```

Comment:
@MercyQueen Can we make false in defaultConfig

Reply:
then this would be disabled for all merchants by default.

Reply:
add it in CommonConfig.purs